packages/frontend/src/components/buttons/BackButton.tsx

```diff
@@ -9,9 +9,10 @@ interface BackButtonProps {
   setView?: (view: number) => void
   url?: string
   resetState?: () => void
+  withBack?: boolean
 }
 
-export function BackButton({ view, setView, url, resetState }: BackButtonProps) {
+export function BackButton({ view, setView, url, resetState, withBack = true }: BackButtonProps) {
```

Comment:
It would be nice to change the default value to false (I think that there are comparable number of places where we are going to use with/without "Back"). That way we could use `<BackButton withBack />` instead `<BackButton withBack={false} />`